rfcs/0130-Implementing Checks API in tc-github while preserving Statuses API

```diff
@@ -0,0 +1,44 @@
+# RFC 130 - Implementing Checks API
+* Comments: [#130](https://api.github.com/repos/taskcluster/taskcluster-rfcs/issues/130)
+* Proposed by: @djmitche
+
+# Summary
+
+We want to implement Checks API without dropping the support os Statuses API. Statuses API would be the default. Our users
```

Comment:
It sounds like the checks api is a general improvement over the statuses api, aside from being independent.

Do we know how many projects are using statuses api programmatically?  The main use I have seen for these apis is the github branch protection feature and the human interface.  I believe branch protection supports the checks api.  Maybe we could open an issue in the repositories which have used taskcluster-github in the last 2-3 months mentioning this change and explaining the changes needed to keep the statuses api enabled.

Reply:
My hope is to make Statuses API the default, so that people won't have to do anything to keep it enabled, and, in fact, so that nobody is even aware of Checks appearance so we could test it in production. After it has been tested and certified fresh, we will let people know that if they want the shiny new Checks, they will have to do this and that to enable them. I plan to write a blog post explaining the feature

Reply:
Ahh, i thought the checks api was ready for primetime, but it sounds more like this is an opt-in testing!  In that case, I'd agree the status api is better as the default.